packages/terra-file-path/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.0.0 - (October 25, 2023)
+
+* Changed
+  * Minor dependency version bump
```

Comment:
If we're doing a full release, can we update the CHANGELOG so it's more reflective of the changes? I don't know what happened here but there are no details in this file at all.

I guess for simplicity, we can update this to:

```suggestion
  * Initial working component release.
```